### PR TITLE
Must-gather: update pre-install.sh script

### PR DESCRIPTION
**Pull request by crombus**
add oc wait cmd and wait for the condition
to be true. make the check function run
in bg and wait for the pids to finish.

Signed-off-by: crombus <pkundra@redhat.com>

### CODE REVIEW

**Review by jarrpa — APPROVED** [review]
/lgtm

File: must-gather/collection-scripts/pre-install.sh [hunk 1]
```diff
@@ -47,36 +47,31 @@ labels(){
...
-                break
-            else
-                pod_name=$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $1}')
-                oc label pod "$pod_name" "${node//./}"-debug='ready'
-                check_debug_pod=true
-            fi
-    done
+    debug_pod_name=$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $1}')
+    # sleep for 60 seconds giving time for debug pod to get created
+    sleep 60
```
Comment (rajatsing) [inline must-gather/collection-scripts/pre-install.sh:52]: just a query, do you think that 60 seconds is enough?. You must have tried it, how was the observation/experience of 60 seconds timeout, is it enough to decide?.

**Review by rajatsing — COMMENTED** [review]
LGTM, got busy with my PR, but jose got this in already!.

File: must-gather/collection-scripts/pre-install.sh (hunk 1 above)
Comment (crombus) [reply must-gather/collection-scripts/pre-install.sh:52]: 60 seconds is just for oc to register the pod. for the condition we are waiting for 200s

**Pull request merged**